app/models/post.rb

```diff
@@ -0,0 +1,18 @@
+class Post < ApplicationRecord
+  belongs_to :user
+  has_many :comments
+  has_many :likes
+  validates :name, presence: true
```

Comment:
- [ ] You should make sure that Title does not exceed 250 characters as well.

Reply:
Added validation for character length🦾.